conf/distro/emlinux-k510.conf

```diff
@@ -2,8 +2,10 @@ require include/emlinux.inc
 
 DISTRO = "emlinux-k510"
 
-LINUX_GIT_BRANCH ?= "linux-5.10.y-cip"
-LINUX_GIT_SRCREV ?= "3ddbe9bf6a006b50d35887b00f96c5768a32b7f3"
+LINUX_GIT_URI = "git://git.kernel.org/pub/scm/linux/kernel/git/stable"
+LINUX_GIT_REPO = "linux.git"
+LINUX_GIT_BRANCH ?= "linux-5.10.y"
+LINUX_GIT_SRCREV ?= "7a1519a74f3d0b06598fb95387688cde41e465d8"
```

Comment:
Could you point to the latest revision in the linux-5.10.y branch and update LINUX_CVE_VERSION?